### DISCUSSION TITLE
how to change time to match system

### DISCUSSION THREAD

**Discussion by tan-wei-xin-alez** [aristocratos/btop#625] [Q&A] [answered]
I'm running `btop version: 1.2.13` and for some reason, the time it is showing differs from my system time as seen below
![image](https://github.com/aristocratos/btop/assets/98415730/b88b5241-da72-40c5-9476-5e94d8f64c15)

I know that menu &rarr; general has a "Clock format" field but I don't see anything to actually change the values themselves

**Comment by deckstose** [comment, marked as answer]
That shouldn't happen regardless of a missing option. Would you mind opening an issue?

**Reply by tan-wei-xin-alez** [reply]
[done](https://github.com/aristocratos/btop/issues/626)
